crates/rune/src/query/mod.rs

```diff
@@ -273,6 +273,17 @@ impl<'a> Query<'a> {
             .modules
             .insert(item.item.clone(), query_mod.clone());
         self.insert_name(&item.item);
+        self.insert_meta(
```

Comment:
There's a method, `CompileVisitor::visit_mod`, that seems to intend to be called when modules are encountered. However, the example in `compiler_docs.rs` doesn't seem to trigger it despite having multiple modules.

I'm unsure what cases it gets called in, but I'm wondering if this change obsoletes the method given that modules will now find themselves caught by `register_meta`.

Reply:
So `visit_mod` is intended to emit spans which "looks like a module". I don't think there's anything more to it. That's why it's being emitted everywhere that module-like expressions are encountered and not just where the module is defined.